buildSrc/src/main/antlr/org/aya/parser/AyaParser.g4

```diff
@@ -63,12 +63,12 @@ fnModifiers : OPAQUE
             | PATTERN_KW
             ;
 
-structDecl : sampleModifiers? (PUBLIC? OPEN)? STRUCT declNameOrInfix tele* type? (EXTENDS idsComma)? (BAR field)* bindBlock?;
+structDecl : sampleModifiers? (PUBLIC? OPEN)? STRUCT declNameOrInfix tele* type? (EXTENDS exprList)? (BAR field)* bindBlock?;
 
 primDecl : PRIM weakId tele* type? ;
 
-field : COERCE? declNameOrInfix tele* type clauses? bindBlock? # fieldDecl
-      | declNameOrInfix tele* type? IMPLIES expr    bindBlock? # fieldImpl
+field : OVERRIDE? COERCE? declNameOrInfix tele* type clauses? bindBlock? # fieldDecl
+      | OVERRIDE? declNameOrInfix tele* type? IMPLIES expr    bindBlock? # fieldImpl
       ;
```

Comment:
These are added but not handled in `AyaProducer`, is it intended?

Reply:
Yes, they will be handled in future prs

Reply:
I see ❤️